### ISSUE TITLE
Noto serif? 

### ISSUE THREAD

**Issue by loquenahak** [zhuchun/cjk#3]
Thank you so much for the font. This is so much productive work. This helped a lot. Are you planning to make a serf font too?

**Comment by zhuchun** [comment]
> Thank you so much for the font. This is so much productive work. This helped a lot. Are you planning to make a serf font too?

No. My past titles allowed people to chat with each other so the CJK characters needed to be covered as many as possible. However, my recent titles are focusing on providing sharp and clear UI font, thus I have no plan on this anymore. 

In case you're interested, the trade off is quite simple, most mobiles nowadays can support 4k texture, and each character requires some pixels on the TMP font texture. Which means the more characters you want to put them in, the smaller font size it could be, otherwise it just can't fit. So we usually use a common Chinese characters(~6k) list to make the TMP font instead of picking up the whole Chinese range from UTF8, that helped us to get a better balance.
